themes/default/content/blog/next-level-iac-package-ecosystems/index.md

```diff
@@ -50,7 +54,7 @@ const fileContents = fs.readFileSync(filePath, {encoding: 'utf-8'});
 // Upload the file to the S3 bucket.
 const bucketObject = new aws.s3.BucketObject("my-bucket-object", {
     bucket: bucket, // Reference to the bucket created above.
-    key: "file.txt", // This will be the file's name in the bucket.
+    key: "file.ext", // This will be the file's name in the bucket.
```

Comment:
Curious why we're changing this from `txt` to `ext`?

Reply:
It was meant to be a pseudo-name, with `.ext` short for extension, as in this is a file with an extention, rather than to imply a specific file with a specific file type.

Reply:
That said, if it's confusing to you, it will probably be confusing to others. I'll change it back to `.txt` everywhere.